src/main/java/core/basesyntax/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransactionParserImpl implements TransactionParser {
+    private static final String SPLIT_DELIMITER = ",";
+    private static final int INDEX_FOR_OPERATION = 0;
+    private static final int INDEX_FOR_NAME = 1;
+    private static final int INDEX_FOR_QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> converterToTransactions(List<String> readData) {
+        List<String[]> collect = readData.stream()
+                .skip(1)
+                .map(s -> s.split(SPLIT_DELIMITER))
+                .toList();
+
+        List<FruitTransaction> transactionList = new ArrayList<>();
+        FruitTransaction fruitTransaction;
+        for (String[] strings : collect) {
+            String fruitName = strings[INDEX_FOR_NAME];
+            int quantity = Integer.parseInt(strings[INDEX_FOR_QUANTITY]);
+            String operationType = strings[INDEX_FOR_OPERATION]
+                    .substring(strings[INDEX_FOR_OPERATION].length() - 1);
```

Comment:
```suggestion
            String operationType = strings[INDEX_FOR_OPERATION];
```
not sure what you need it for